src/common/src/system_param/mod.rs

```diff
@@ -91,7 +91,7 @@ macro_rules! for_all_params {
             { enable_tracing,                           bool,                           Some(false),                    true,   "Whether to enable distributed tracing.", },
             { use_new_object_prefix_strategy,           bool,                           None,                           false,  "Whether to split object prefix.", },
             { license_key,                              risingwave_license::LicenseKey, Some(Default::default()),       true,   "The license key to activate enterprise features.", },
-            { time_travel_retention_ms,                 u64,                            Some(0_u64),                    true,   "The data retention period for time travel, where 0 indicates that it's disabled.", },
+            { time_travel_retention_ms,                 u64,                            Some(600000_u64),              true,   "The data retention period for time travel.", },
```

Comment:
IIUC, old cluster won't use the new default value after upgrade. I think we need to have logic to overwrite 0 `time_travel_retention_ms` to the new default forcibly.